atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -33,6 +33,7 @@ class FetchSourcesPlugin(PreBuildPlugin):
     key = PLUGIN_FETCH_SOURCES_KEY
     is_allowed_to_fail = False
     DOWNLOAD_DIR = 'image_sources'
+    DOWNLOAD_REMOTE_DIR = 'remote_sources'
```

Comment:
Should we rename these 2 dirs and vars to better reflect what they are?

Maybe something like SRPMS_DOWNLOAD_DIR, REMOTE_SOURCES_DOWNLOAD_DIR and srpms instead of image_sources

Reply:
sure